examples/crowd-funding/src/contract.rs

```diff
@@ -389,7 +389,8 @@ impl CrowdFunding {
         Ok(())
     }
 
-    fn initialization_argument(&self) -> &InitializationArgument {
+    // Trailing underscore to avoid conflict with the generated GraphQL function.
```

Comment:
We can, I was just trying to minimize `async-graphql`-specific magic in our Linera examples :)

Reply:
Will do.  Since the actual name of the function doesn't matter for `async-graphql` it might be nice if `async-graphql` generated functions called `__async_graphql_initialization_argument` or something so they won't conflict…